config/routes.rb

```diff
@@ -2,5 +2,13 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "users#index"
+  get "/users/:id", to: "users#show"
+  get "users/:id/posts", to: "posts#index"
+  get "users/:id/posts/:id", to: "posts#show"
+
```

Comment:
- [ ] It would be nice if you make a dedicated route to get all the users because i can see that root route is set on getting all the users but to follow the project requirements you should create routes for all the methods 👍
- [ ] It would be great if you make your routes as below to follow the new sytanx 👍

```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```